code/game/objects/items/robot/robot_items.dm

```diff
@@ -120,12 +120,14 @@
 			if(!shock_cooldown)
 				if(ishuman(M)||ismonkey(M))
 					M.electrocute_act(15, user, 1)
+					add_logs(user, M, "eletrocuted with \the [src]", admin = (user.ckey && M.ckey) ? TRUE : FALSE)
```

Comment:
Aren't they needed to avoid reporting to admins things that are happening to mobs with no ckey?